src/components/inputs/money-input/money-input.spec.js

```diff
@@ -338,11 +338,11 @@ describe('MoneyInput', () => {
   });
 
   it('should show the passed value', () => {
-    const { getByLabelText } = render(
+    const { getByLabelText, getByTestId } = render(
```

Comment:
Look at these beautiful changes of the spec. This PR completely ripped out `downshift` and replaced it with `react-select`. This is a fundamental change in the implementation - and yet the tests remain mostly the same.

Even though I did all the changes above, this spec was able to find a mistake: The select dropdown wasn't usable with a label - because I was passing `id` instead of `inputId` to `Select`. This is so impressive. Testing the DOM instead of the implementation/React components is such an excellent idea.

This is why I love `react-testing-library`.